README.md

```diff
@@ -1,8 +1,8 @@
 [ ![Codeship Status for databraid-dashboard/transit-api](https://app.codeship.com/projects/459d8c80-5ead-0135-6b43-16bdadfc7181/status?branch=master)](https://app.codeship.com/projects/238758)
 
-# Databraid API Starter Repo Without Postgres
+# Description
```

Comment:
@Sanderson239 please make this an H2:

`## Description`

And add an H1 above it with `# Databraid Transit API`